src/main/java/com/bzabek/ticketbookingapp/entities/SeatReserved.java

```diff
@@ -0,0 +1,83 @@
+package com.bzabek.ticketbookingapp.entities;
+
+import javax.persistence.*;
+import java.util.UUID;
+
+@Entity
+public class SeatReserved {
+
+    @Id
+    @GeneratedValue
+    private UUID id;
+    private boolean reserved;
+    @OneToOne(mappedBy = "seatReserved")
+    private Reservation reservation;
+    @ManyToOne
+    private Screening screening;
+    private Long rowNr;
```

Comment:
czy ta i poniższa wartość może być null?

Reply:
Jest to możliwe. Foreign key może przyjmować wartość null. Raczej nie rekomendowałbym aby to robić. Nullowe wartości mogą prowadzić do powstawania błędów.
W swojej aplikacji nie dopuszczam do przypadku aby FK miał wartość null.